README.md

```diff
@@ -144,6 +150,18 @@ All commands should be run from `workbench/api`
 
 While the API is running locally, saving a .java file should cause a recompile and reload of that class. Status is logged to the console. Not all changes reload correctly (e.g., model classes do not appear to reload more than once).
 
+#### Caveats
+
+The first time a server is started after a database clear (`./project.rb docker-clean`, `./project.rb run-local-all-migrations`), it may take some time for a billing project to become available for workspace creation. Generally, this takes about 5 minutes, but has been known to take upwards of 40 in some cases. The billing project buffer size is set to 2 for local environments (`config_local.json`). A cron job runs every minute while the API server is running to check if there are empty slots.
+
+To check whether you have available billing projects:
```

Comment:
nit: to me, `available` indicates that the following query will find entries that are ready to be picked up but it's actually just finding ALL entries.
 
I would just drop that adjective or add `where status=2` to the query.